cpp/src/strings/regex/regcomp.cpp

```diff
@@ -14,10 +14,11 @@
 * limitations under the License.
 */
 
+#include <cudf/utilities/error.hpp>
 #include <strings/regex/regcomp.h>
 
 #include <string.h>
-
+#include <map>
```

Comment:
From the below it looks like you meant set?

```suggestion
#include <set>
```